src/ui/components/AddBounty.tsx

```diff
@@ -15,12 +15,14 @@ import {BN, bnToHex} from '@polkadot/util';
 import {formattedStringToBn} from 'src/utils/balance';
 import {countUtf8Bytes} from 'src/utils';
 import {useStartTx} from 'context/TxContext';
+import {BountiesQueryResult} from 'src/api/hooks/useBounties';
 
 type Props = {
   onClose: () => void;
+  onSubmit: () => BountiesQueryResult;
```

Comment:
This is the same as having `refetch` as prop name, we cannot pass anything that is not a function that returns a query result if in the future we want to add something. Just keep the `() => void` as you had before